rest_framework/pagination.py

```diff
@@ -534,7 +540,7 @@ def get_next_link(self):
         if not self.has_next:
             return None
 
-        if self.cursor and self.cursor.reverse and self.cursor.offset != 0:
+        if self.cursor and self.cursor.reverse and self.cursor.offset != 0 and self.page:
```

Comment:
> tell me what both are unacceptable and then accept both, again?

Also because I haven't reached a judgement yet on which would be acceptable, and it'll be much easier to make that if they're properly separated.